packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts

```diff
@@ -447,6 +448,9 @@ export class SearchableModelTransformer extends TransformerPluginBase {
     // but if mappings are defined this will ensure the mapping is also applied to the search results
     for (const def of this.searchableObjectTypeDefinitions) {
       const modelName = def.node.name.value;
+      if (isRDSModel(ctx as TransformerContextProvider, modelName)) {
```

Comment:
I'm assuming we will not support `@searchable` for RDS at all--should we throw an error rather than silently ignoring?

Reply:
Converted this to an Error and added an unit test. We actually have a check for unsupported directives even before the transformers are invoked and a plan to move it into the transfotmers before release. So, this is one step towards it.